syllabustool/settings.py

```diff
@@ -27,11 +27,19 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config('DEBUG', default=False, cast=bool)
 
-EMAIL_HOST = config('EMAIL_HOST', default='localhost')
-EMAIL_PORT = config('EMAIL_PORT', default=25, cast=int)
-EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD', default='')
-EMAIL_HOST_USER = config('EMAIL_HOST_USER', default='')
-EMAIL_USE_TLS = config('EMAIL_USE_TLS', default=False, cast=bool)
+EMAIL_BACKEND ="django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = '587'
+EMAIL_HOST_PASSWORD = 'Znqbdyq8'
```

Comment:
This data is supposed to be stored in the .env file and pulled from there with the deleted lines.  Our app's password is now commited to the repo and is vulnerable.  It should be changed.

Reply:
ill make a small story to change this next iteration as i wasnt sure if this was a big deal for the sake of our project and this was done early on in the iteration and i forgot about it but ill change the password now and update the code for in the coming iteration as our server isnt live this is fine to wait